leftwm/src/bin/lefthk-worker.rs

```diff
@@ -3,8 +3,14 @@ use lefthk_core::{config::Config, worker::Worker};
 #[cfg(feature = "lefthk")]
 use xdg::BaseDirectories;
 
-#[cfg(feature = "lefthk")]
 fn main() {
+    // we need this little shenanigan to allow actually building with `--no-default-features`
+    #[cfg(feature = "lefthk")]
+    lefthk_worker_main();
+}
+
+#[cfg(feature = "lefthk")]
+fn lefthk_worker_main() {
```

Comment:
Isn't that the same as just removing `#[cfg(feature = "lefthk")]` above the `fn main()`?

Reply:
Thanks this was exactly the hint I needed to finde a better solution.